packages/core/variables/_font-size.scss

```diff
@@ -26,142 +26,87 @@ $line-height-6: rem(48px);
 $line-height-7: rem(56px);
 $line-height-8: rem(72px);
 
-@mixin h1-heading {
-    @include h1-heading--desktop;
-    @include small-device {
-        @include h1-heading--mobile;
+$text-styles: (
+    "desktop/title-large": (
+        font-size: $font-size-12,
+        line-height: $line-height-8,
+        font-weight: $normal-weight,
+    ),
+    "compact/title-large": (
+        font-size: $font-size-10,
+        line-height: $line-height-6,
+        font-weight: $normal-weight,
+    ),
+    "desktop/title-small": (
+        font-size: $font-size-11,
+        line-height: $line-height-7,
+        font-weight: $normal-weight,
+    ),
+    "compact/title-small": (
+        font-size: $font-size-9,
+        line-height: $line-height-5,
+        font-weight: $normal-weight,
+    ),
+    "desktop/heading-large": (
+        font-size: $font-size-9,
+        line-height: $line-height-5,
+    ),
+    "compact/heading-large": (
+        font-size: $font-size-7,
+        line-height: $line-height-4,
+    ),
+    "desktop/heading-medium": (
+        font-size: $font-size-7,
+        line-height: $line-height-4,
+    ),
+    "compact/heading-medium": (
+        font-size: $font-size-5,
+        line-height: $line-height-3,
+    ),
+    "desktop/heading-small": (
+        font-size: $font-size-5,
+        line-height: $line-height-3,
+    ),
+    "compact/heading-small": (
+        font-size: $font-size-3,
+        line-height: $line-height-2,
+    ),
+    "desktop/lead": (
+        font-size: $font-size-8,
+        line-height: $line-height-5,
+    ),
+    "compact/lead": (
+        font-size: $font-size-6,
+        line-height: $line-height-4,
+    ),
+    "desktop/body": (
+        font-size: $font-size-4,
+        line-height: $line-height-4,
+    ),
+    "compact/body": (
+        font-size: $font-size-3,
+        line-height: $line-height-2,
+    ),
+    "desktop/small": (
+        font-size: $font-size-2,
+        line-height: $line-height-2,
+    ),
+    "compact/small": (
+        font-size: $font-size-2,
+        line-height: $line-height-2,
+    ),
+    "desktop/micro": (
+        font-size: $font-size-1,
+        line-height: $line-height-1,
+    ),
+    "compact/micro": (
+        font-size: $font-size-1,
+        line-height: $line-height-1,
+    ),
```

Comment:
Disse er like i `desktop`- og `compact`-variantene, men jeg har tatt med begge for å sikre samsvar med Figma.

Reply:
Iom. at denne endringen definerer at små enheter er `compact`. Vil det da være riktig å endre fra `small-device` til `compact-device` også? 

Et naturlig spørsmål er hvilke skjermstørrelser som skal defineres. Er en 7" tablet compact? Er en 4k skjerm desktop?

Reply:
Vi må helt klart gå opp skjermstørrelsene vi benytter oss av ordentlig. Litt av grunnen til å endre fra `mobile` til `compact` er imidlertid å fjerne sammenkoblingen med skjermstørrelse. Selv om `compact`-versionene brukes på små skjermer er også tanken at de kan brukes i interne løsninger med høy informasjonstetthet.

Reply:
Høres fornuftig ut 👍